Properly render 'expand' and 'collapse' action on notification

martin-fleck-at opened the pull request:
#### What it does
- Fix assignment of class names
- Scale icons to better fit the 'close icon'
-- Remove hover background as due to the scaling it looks weird
-- Use scaling as indicator instead

![fix_expandCollapse](https://user-images.githubusercontent.com/19170971/143581539-055a536c-25ed-4812-8ecb-b0dfd966391b.gif)

Fixes https://github.com/eclipse-theia/theia/issues/10470

#### How to test

1. Produce a message over 500 characters (for example through a sample menu contribution)
2. See that the expand/collapse action is there and working

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

vince-fugnitto commented on packages/messages/src/browser/style/notifications.css:
```diff
@@ -191,17 +191,27 @@
     margin-left: 4px;
 }
 
+.theia-notification-actions > li:hover {
+    /* remove default background change as due to the different scaling of the items it might look odd */ 
+    background-color: transparent;
+    transform: scale(1.5);
+}
```
@martin-fleck-at we should neither remove the background, or change the scaling of the icons when hovering as it is the old way of rendering icons, and does not align with the rest of the icon buttons or toolbar items present in the framework. 

It currently breaks the updates from:
- https://github.com/eclipse-theia/theia/pull/9864
- https://github.com/eclipse-theia/theia/pull/10078

martin-fleck-at commented:
@vince-fugnitto Thank you for your fast review! I removed the adaptation again and just reduced the scaling a bit to make the icon size match the close icon a bit better. If you want I can remove that as well though:

![fix_expandCollapse2](https://user-images.githubusercontent.com/19170971/143595955-1ac883cc-e677-459a-8c1a-15d20bc83782.gif)

vince-fugnitto commented:
@martin-fleck-at I haven't tested your latest changes yet, but based on the gif you sent something still looks incorrect. The behavior should be like the `close` icon, and align with vscode:

https://user-images.githubusercontent.com/40359487/143596448-fd6a7b55-659a-461e-9c1e-04587a7a6af2.mp4

martin-fleck-at commented:
@vince-fugnitto Got it! I updated the change but I removed the scaling now because otherwise it looks weird:

![fix_noScale](https://user-images.githubusercontent.com/19170971/143599714-01ef1c7e-762d-4d2e-8cad-2d56c2baa169.gif)
(without scaling and submitted)

![fix_scale](https://user-images.githubusercontent.com/19170971/143599869-37c80e57-a3c7-4d15-924f-d41011274e45.gif)
(with the scaling defined in master)

msujew reviewed (commented):
Great, thanks for taking care of that.

Something didn't seem quite right about the direction of the chevron both in expanded and collapsed form. I took a look at how VSCode handles this. Seems like my intuition proved correct :)

Collapsed:

![image](https://user-images.githubusercontent.com/4377073/143869042-86f58b01-55a2-403a-befa-3889b7033669.png)

Expanded:

![image](https://user-images.githubusercontent.com/4377073/143869085-22347a4a-6a08-40de-aaf7-5daf00be5f68.png)

Do you mind adjusting the rotation of the chevron to align with VSCode? Afterwards, this looks good to go.

martin-fleck-at commented:
@msujew Very good catch! Thank you! I aligned the chevron direction accordingly:

![arrow](https://user-images.githubusercontent.com/19170971/144006823-42f1a193-cd25-429c-948c-ebe58e7f54fb.gif)

msujew reviewed (approved):
Thanks, looks great!

- [x] The icon is visible again after your changes. (compared to master)
- [x] The chevron does not increase its size on hover/click.
- [x] The chevron rotation is aligned with vscode for expansion/collapse/hover.

msujew commented on packages/messages/src/browser/style/notifications.css:
```diff
@@ -192,17 +192,11 @@
-.theia-notification-actions > .collapse:active {
-    transform: rotate(-90deg) scale(1.8);
+    transform: rotate(0deg);
 }
```
Minor: We don't need that rule anymore (rotating by 0deg does nothing) .

martin-fleck-at replied:
:man_facepalming: I removed the rule, thanks!
